osf/models/spam.py

```diff
@@ -200,6 +200,7 @@ def confirm_spam(self, domains=None, save=False, train_akismet=True):
         if domains:
             if 'domains' in self.spam_data:
                 self.spam_data['domains'].extend(domains)
+                self.spam_data['domains'] = list(set(self.spam_data['domains']))
```

Comment:
Torn between this approach and updating the domain extraction to only send the domain if the reference was created. But this is probably safer in case something else winds up passing these domains.

Reply:
(this also has the added benefit of deduping existing things)